lib/src/kbasesearchengine/search/ObjectData.java

```diff
@@ -526,5 +548,18 @@ public Builder withKeyProperty(final String key, final String property) {
             keyProps.put(key, property);
             return this;
         }
+
+        /** Adds the highlight fields to the object.
+         * @param highlight the map of fields returned from elasticsearch.
+         * @return this builder.
+         */
+        public Builder withHighlight(final String field, final ArrayList highlight) {
+            Utils.notNullOrEmpty(field, "field cannot be null or whitespace");
+            Utils.noNulls(highlight, "highlight cannot be null");
+            if(highlight.size() > 0) {
```

Comment:
I don't see a test for this path

Reply:
buildMaximal in ObjectDataTest tests it?

Reply:
It doesn't test the case that highlight.size() < 1 as far as I can tell